.github/workflows/on_push.yaml

```diff
@@ -2,21 +2,23 @@ name: On Push
 on:
   push:
     branches-ignore:
-      - "staging"
-      - "master"
+      - 'staging'
+      - 'master'
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Install node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          cache: "yarn"
-          node-version: 16
-          registry-url: "https://registry.npmjs.org"
+          cache: 'yarn'
+          node-version: 18
+          registry-url: 'https://registry.npmjs.org'
       - name: Install deps
         run: yarn
+      - name: Optimize SVG images
+        run: yarn optimize-svg
```

Comment:
if i understand correctly this command will create a change that need to be committed,,,
which means, we should have a flow to commit these changes
i would say it is not needed

Reply:
yes, it can be done locally each time a change has been made in the drawio.svg. 
However, I added it here so when an update in the OSX/SDK is done, everything is resolved automatically. 
(let's say someone makes some changes in osx repo drawio.svg, he merge them so a PR is triggered here in the dev-portal, someone should before merging the dev-portal auto-created PR enter in that auto-created branch and run the command to optimize svg files push the changes and then merge the branch. So it won't be a fully automated process as it is now.)
That said, that will happen hardly ever I guess.